packages/dai-plugin-mcd/contracts/abiMap.js

```diff
@@ -14,7 +14,9 @@ module.exports = {
   MCD_ADM: require('./abis/DSRoles.json'),
   MCD_CAT: require('./abis/Cat.json'),
   OLD_MCD_CAT: require('./abis/Cat.json'),
+  'MCD_CLIP_*': require('./abis/Clipper.json'),
   MCD_DAI: require('./abis/DSToken.json'),
+  MCD_DOG: require('./abis/Dog.json'),
```

Comment:
These are for `mcd` plugin so just want to make sure we want to include them here. Same with all the other`mcd` plugin abi updates below.

Reply:
removed, not needed.